packages/bits/tsconfig.lib.locl.json

```diff
@@ -1,6 +1,3 @@
 {
-    "angularCompilerOptions": {
-        "enableIvy": true
```

Comment:
I imagine ivy is enabled by default now?

Reply:
yes